RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -23,6 +23,9 @@
 from RecoJets.JetProducers.fixedGridRhoProducerFastjet_cfi import *
 fixedGridRhoFastjetAllTmp = fixedGridRhoFastjetAll.clone(pfCandidatesTag = cms.InputTag("particleFlowTmp"))
 
+from RecoJets.JetProducers.ak4PFJets_cfi import *
+ak4PFJetsTmp = ak4PFJets.clone(src = cms.InputTag("particleFlowTmp"))
```

Comment:
Yes that's fine, the pfpileup instances which use particleFlowTmp are running as part of reco, so using a transient jet collection should be fine.